### ISSUE TITLE
Changes in Pug layout files not updating (with `lume --serve`)

### ISSUE THREAD

**Issue by yet-another-dev** [lumeland/lume#39]
Example:

```pug
//- _layout.pug

html.no-js(lang="en")
   head
      meta(charset='utf-8')
   body
      div A message from `_layout.pug`
      block pageContent


//- home.pug

extends _layout.pug
block pageContent
   h1 Home
```

Making a change in `home.pug` triggers an update. Making a change in `_layout.pug` does _not_ trigger an update.

When making a change in `_layout.pug`, the cli does say "Changes detected. Building... Done", but the browser does not reload and Lume doesn't seem to actually rebuild (manually reloading the browser does not show the update).

**Comment by oscarotero** [comment]
Mmm, ok. Looks like pug caches the layouts loaded with `extends`. I'll try to fix it.
Meanwhile, you can use a frontmatter to configure your layout. Look this example: https://github.com/lumeland/test/blob/master/pug/pug-example.pug

**Comment by oscarotero** [comment]
Ok, this bug is fixed in `0.15.1`.
